Rakefile

```diff
@@ -41,8 +41,13 @@ task :web do
 
   require 'sidekiq/web'
   require 'sidecloq/web'
+  require 'securerandom'
 
   Rack::Server.start(
-    app: Sidekiq::Web
+    app:
+      Rack::Builder.app do
+        use Rack::Session::Cookie, secret: SecureRandom.hex(32), same_site: true, max_age: 86400
```

Comment:
Technically since this just for development, it doesn't have to be a securerandom value and could just be hardcoded to anything like `"0123456789abcdef"`, but I felt it was more responsible to use a "proper" example should someone look here for inspiration.  Even so, I don't feel super strongly about it, so it's up to you.